src/functions.ts

```diff
@@ -1,45 +1,74 @@
-// Module
-export const users = require("../models/data.js");
-const { writeFileSync } = require("fs");
+// model
+import userModel from "../models/user";
 
-interface userz {
-  id: number;
-  fname: string;
-  lname: string;
-}
+//Display all Users
+export const displayUsers = () => {
+  return userModel
+    .find({})
+    .then((value) => {
+      return value;
+    })
+    .catch((error) => {
+      return error;
+    });
+};
 
 // Find User
-export const findUser = (id: number) => {
-  const singleUser = users.find((user: userz) => user.id === id);
-  return singleUser;
+export const findUser = async (userid: number) => {
+  return await userModel
```

Comment:
When returning a result from an async function within an async function, you can omit the `await`.

Example:
```
async () => {
  return anAsyncFunction();
}